includes/Plugin.php

```diff
@@ -140,6 +142,20 @@ class Plugin {
 	 */
 	public $experiments;
 
+	/**
+	 * AMP plugin integration.
+	 *
+	 * @var AMP
+	 */
+	public $amp;
+
+	/**
+	 * Jetpack integration.
+	 *
+	 * @var Jetpack
+	 */
+	public $jetpack;
```

Comment:
PHPMD rightfully complains that the `Plugin` class now has a lot of public fields.

Sounds like a refactor is due here.

Reply:
Let's ignore for now and follow up in another ticket 👍